go/internal/demo/endpoint.go

```diff
@@ -59,6 +60,11 @@ func (e endpoint) init() (*echo.Echo, error) {
 		api.GET("/access-token", e.accessToken)
 	}
 
+	r.HTTPErrorHandler = func(err error, context echo.Context) {
+		log.Errorf("error processing %s : %+v", context.Path(), err)
```

Comment:
can you use the internal log package here? this is a sample so we want to keep it as clean and neat as possible, I don't think we need "github.com/labstack/gommon/log" package

Reply:
Ah I thought I was using the default one. I've updated to use the echo logger, think it should be simple as we're already using the echo framework.